packages/salesforcedx-vscode-replay-debugger/src/core/logFileUtil.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import { EOL } from 'os';
+import { DebugLogEvent, LogEntry, NoOp } from '../events';
+
+export class LogFileUtil {
```

Comment:
Something to keep in mind. Eventually, we want to be able to operate on logs that come back as a stream. So, let's not hardcode and assume that we have the entire contents in memory at once.